CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 0.3.0 - 2020-07-31
+
+### Added
+- Folder structure will be displayed for the app and api location steps when creating static web apps
+- Display GitHub Actions in tree
```

Comment:
These three are all a context menu actions in the tree, yet you formatted them all differently
- Rerun and cancel actions from the tree
- Edit configuration file
- `Delete Environment...` context menu on environments

Can we do something more consistent? Maybe just:
- Rerun and cancel actions
- Edit configuration file
- Delete Environment